src/components/selector_menu/list.js

```diff
@@ -1,26 +1,27 @@
-var React = window.React || require('react/addons');
+import React from 'react';
 
-var List = React.createClass({
+const List = React.createClass({
 
   propTypes: {
     optionNames: React.PropTypes.array,
     onOptionSelect: React.PropTypes.func.isRequired
   },
 
-  getDefaultProps: function() {
+  getDefaultProps() {
     return {
       optionNames: []
     };
   },
 
-  render: function() {
-    var options = this.props.optionNames.map(function(optionName) {
-      return optionName.length ? (
-        <li key={optionName} className='option' onClick={_.partial(this.props.onOptionSelect, optionName)}>
-          <span className='inner'>{optionName}</span>
+  render() {
+    let options = this.props.optionNames.map((name) => {
```

Comment:
Not opposed to changing this, but just in case you had a perf concern "cached state in render" not really a thing https://github.com/planningcenter/react-patterns/issues/22